utils/iam_dataset.py

```diff
@@ -258,7 +252,20 @@ def _get_bb_of_item(self, item, height, width):
         x2 = float(x2) / width
         y1 = float(y1) / height
         y2 = float(y2) / height
-        return [x1, y1, x2 - x1, y2 - y1]
+        bb = [x1, y1, x2 - x1, y2 - y1]
+
+        # Expand the bounding box by 3% to relax the boundaries
+        s = 0.03
```

Comment:
This shouldn't be here IMO it should be part of the transform, and could you make it parametric?